src/App.js

```diff
@@ -1,83 +1,77 @@
-import React from "react";
-import "./App.css";
+import React, { useEffect, useState } from 'react';
+import './App.css';
+import { getFilteredForecasts } from './utils/utils';
+import { cities } from './utils/cities.js';
+
+import CurrentWeather from './components/CurrentWeather/CurrentWeather';
+import Forecast from './components/Forecast/Forecast';
+import Spinner from './components/Spinner/Spinner';
 
 const App = () => {
+  const API_ID = process.env.REACT_APP_MY_API_ID;
+
+  const [weather, setWeather] = useState(null);
+  const [city, setCity] = useState('Prague');
+  const [forecasts, setForecasts] = useState(null);
+
+  const fetchWeather = (city) => {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&APPID=${API_ID}`,
+    )
+      .then((response) => response.json())
+      .then((json) => setWeather(json));
+  };
+
+  const fetchForecasts = (city) => {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=metric&appid=${API_ID}`,
+    )
+      .then((response) => response.json())
+      .then((json) => setForecasts(getFilteredForecasts(json.list)));
+  };
+
+  useEffect(() => {
+    fetchWeather(city);
+    fetchForecasts(city);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Bez disablování mi linter i konzole u [cities] házeli warning, že 'react hook useEffect has missing dependencies'. Zkoušela jsem ty fetch funkce a API_ID předat jako parametr, ale potom to zase upozorňuje, že se při každém zavolání useEffectu přepisují. Někde jsem našla jako možné řešení přesunout konstantu API_ID a fetch funkce přímo dovnitř useEffectu (zkoušela jsem to, fungovalo to, ale pak jsem to vrátila, když jsem zkoušela, jestli to nevyřeší problém s netlify, ale ten jsem vyřešila jinak). Měla bych to teda dát dovnitř? Pokud bys mi k tomu mohla napsat víc, budu ráda :)